main_code/Email/inbox-dialogflow.py

```diff
@@ -0,0 +1,373 @@
+'''
+Two credential files required. For dialogflow & Gmail
+
+References:
+Getting started with Authentication (GCP)
+https://cloud.google.com/docs/authentication/getting-started#auth-cloud-implicit-python
+
+Set path
+https://stackoverflow.com/questions/45501082/set-google-application-credentials-in-python-project-to-use-google-api
+
+Gmail API SCOPES https://developers.google.com/gmail/api/auth/scopes
+'''
+

```

Comment:
Additionally: make sure we're using all of these things, and no unnecessary packages are imported